### PR TITLE
gplates: 2.3.0 -> 2.4

### LINKED ISSUE #275452: Update request: gplates 2.3 → 2.4
- Package name: gplates
- Latest released version: 2.4
- Current version on the unstable channel: 2.3
- Current version on the stable/release channel: 2.3
- [X] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**

> This package has no maintainers.

https://www.gplates.org/news/2023-12-08-GPlates-2-4-released/  
https://github.com/GPlates/GPlates/releases/tag/GPlates-2.4

-----

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by arthsmn**
## Description of changes

Changelog: https://www.gplates.org/news/2023-12-08-GPlates-2-4-released/
Closes: https://github.com/NixOS/nixpkgs/issues/275452

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/science/misc/gplates/default.nix [hunk 1]
```diff
@@ -17,6 +16,7 @@
 , python3
 , qtxmlpatterns
 , qwt
+, qt5
```
Comment (wegank) [inline pkgs/applications/science/misc/gplates/default.nix:19]:
```suggestion
, wrapQtAppsHook
```

File: pkgs/applications/science/misc/gplates/default.nix [hunk 2]
```diff
@@ -48,6 +48,7 @@ in mkDerivation rec {
     cmake
     doxygen
     graphviz
+    qt5.wrapQtAppsHook
```
Comment (wegank) [inline pkgs/applications/science/misc/gplates/default.nix:51]:
```suggestion
    wrapQtAppsHook
```

File: pkgs/applications/science/misc/gplates/default.nix (hunk 1 above)
Comment (arthsmn) [reply pkgs/applications/science/misc/gplates/default.nix]: Done.

File: pkgs/applications/science/misc/gplates/default.nix (hunk 2 above)
Comment (arthsmn) [reply pkgs/applications/science/misc/gplates/default.nix]: Done.

**Review by wegank — DISMISSED** [review]

**Comment by wegank** [comment]
Result of `nixpkgs-review pr 278165` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gplates</li>
  </ul>
</details>

**Review by wegank — CHANGES_REQUESTED** [review]
`gplates` still reports itself as version 2.3.0, please update the url and the hash.

**Comment by arthsmn** [comment]
> `gplates` still reports itself as version 2.3.0, please update the url and the hash.

I'm using fetchFromGitHub and also using the finalAttrs.version in it.

**Review by wegank — APPROVED** [review]

**Pull request merged**